packages/amplify-dotnet-function-runtime-provider/src/utils/detect.ts

```diff
@@ -42,23 +42,26 @@ export const detectDotNetCore = async (): Promise<CheckDependenciesResult> => {
     }
   }
 
-  if (!toolInstalled) {
-    toolInstalled = await installGlobalTool('Amazon.Lambda.Tools');
-  }
-  if (!testToolInstalled) {
-    testToolInstalled = await installGlobalTool('Amazon.Lambda.TestTool-3.1');
-  }
-
   // Verify that a dotnet 3.1 SDK and the dotnet Lambda tools is installed locally
   if (sdkInstalled && toolInstalled && testToolInstalled) {
     return {
       hasRequiredDependencies: true,
     };
   } else {
-    return {
+    const result = {
       hasRequiredDependencies: false,
-      errorMessage: `Expected ${executableName} minimum version ${currentSupportedVersion}, but found: ${installedSdks}`,
+      errorMessage: 'Unable to detect required dependencies:\n',
     };
+    if (!sdkInstalled) {
+      result.errorMessage += '- The .NET Core 3.1 SDK must be installed.\n';
+    }
+    if (!toolInstalled) {
+      result.errorMessage += '- The Amazon.Lambda.Tools global tool must be installed.\n';
+    }
+    if (!testToolInstalled) {
+      result.errorMessage += '- The Amazon.Lambda.TestTool-3.1 global tool must be installed.\n';
```

Comment:
Please add: 'Install by running: dotnet tool install -g Amazon.Lambda.TestTool-3.1'

Reply:
Also the installation of these tools must be added after dotnet install: https://github.com/aws-amplify/amplify-cli/blob/master/.circleci/config.base.yml#L185

Reply:
Ah, OK! I don't have access to CCI, so I wasn't even aware of this file. Adding now...